src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -101,26 +128,20 @@ class MetaInfoPanelImpl extends React.PureComponent<Props> {
 
     let cpuCount = null;
     if (meta.physicalCPUs || meta.logicalCPUs) {
-      let physicalCPUs = null;
-      let logicalCPUs = null;
-      if (meta.physicalCPUs) {
-        physicalCPUs =
-          meta.physicalCPUs +
-          ' physical ' +
-          (meta.physicalCPUs === 1 ? 'core' : 'cores');
-      }
-      if (meta.logicalCPUs) {
-        logicalCPUs =
-          meta.logicalCPUs +
-          ' logical ' +
-          (meta.logicalCPUs === 1 ? 'core' : 'cores');
-      }
       cpuCount = (
         <div className="metaInfoRow">
-          <span className="metaInfoLabel">CPU:</span>
-          {physicalCPUs}
-          {physicalCPUs && logicalCPUs ? ', ' : null}
-          {logicalCPUs}
+          <span className="metaInfoLabel">
+            <Localized id="MenuButtons--metaInfo--cpu">CPU:</Localized>
+          </span>
+          <Localized
+            id="MenuButtons--metaInfo--physical-cpu"
+            vars={{ physicalCPUs: meta.physicalCPUs }}
+          />
+          {meta.physicalCPUs && meta.physicalCPUs ? ', ' : null}
+          <Localized
+            id="MenuButtons--metaInfo--logical-cpu"
+            vars={{ logicalCPUs: meta.logicalCPUs }}
+          />
```

Comment:
I think this needs to be a bit different: indeed this is a full sentence and needs to be localized completely.
Indeed the localizers may need to translate differently the full sentence, so we can't do a concatenation like this.

So this should be something like this:
```js
let cpuCount = null;
if (meta.physicalCPUs && meta.logicalCPUs) {
  cpuCount = <Localized
    id="MenuButtons--metaInfo--physical-and-logical-cpu"
    vars={{  physicalCPUs: meta.physicalCPUs, logicalCPUs: meta.logicalCPUs }}
  />
} else if (meta.physicalCPUs) {
  cpuCount = <Localized
    id="MenuButtons--metaInfo--physical-cpu"
    vars={{  physicalCPUs: meta.physicalCPUs }}
  />
} else if (meta.logicalCPUs) {
  cpuCount = <Localized
    id="MenuButtons--metaInfo--logical-cpu"
    vars={{  logicalCPUs: meta.logicalCPUs }}
  />
}

... Below, where we use cpuCount:
{cpuCount ? (
  <div className="metaInfoRow">
    <span className="metaInfoLabel">
      <Localized id="MenuButtons--metaInfo--cpu">CPU:</Localized>
    </span>
    {cpuCount}
</div>) : null}
```

What do you think?

Reply:
Yes this looks neat, I have made the changes.